File: packages/expo-keep-awake/android/src/main/java/expo/modules/keepawake/KeepAwakePackage.java
package expo.modules.keepawake;

import android.content.Context;

import expo.modules.core.ExportedModule;
import expo.modules.core.interfaces.InternalModule;
import expo.modules.core.interfaces.Package;

import java.util.Collections;
import java.util.List;

public class KeepAwakePackage implements Package {

  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.<ExportedModule>singletonList(new KeepAwakeModule(context));
  }

  @Override
  public List<? extends InternalModule> createInternalModules(Context context) {
    return Collections.singletonList(new ExpoKeepAwakeManager());
  }
}

File: packages/expo-keep-awake/android/src/main/java/expo/modules/keepawake/KeepAwakePackage.kt
package expo.modules.keepawake

import android.content.Context

import expo.modules.core.interfaces.InternalModule
import expo.modules.core.interfaces.Package

class KeepAwakePackage : Package {
  override fun createInternalModules(context: Context): List<InternalModule> {
    return listOf(ExpoKeepAwakeManager())
  }
}
